core/src/main/scripts/importer/cbioportal_common.py

```diff
@@ -222,7 +234,8 @@ class MetaFileTypes(object):
     "org.mskcc.cbio.portal.scripts.ImportGisticData" : False,
     "org.mskcc.cbio.portal.scripts.ImportMutSigData" : False,
     "org.mskcc.cbio.portal.scripts.ImportProfileData" : True,
-    "org.mskcc.cbio.portal.scripts.ImportTimelineData" : True
+    "org.mskcc.cbio.portal.scripts.ImportTimelineData" : True,
+    "org.mskcc.cbio.portal.scripts.ImportGenePanelProfileMap" : True
```

Comment:
Ah, it looks like this logic is not being tested when using the deprecated `-c import-study-data -m <metadata_file> -d <data_file>` form instead of the `-s <study_directory>` option.